browser/ui/brave_scheme_utils_unittest.cc

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
```

Comment:
this file is not added to `GN`, presubmit rightfully complains. You can run locally `npm run presubmit` to see the error.

Reply:
I think it got missed in recent merge conflict, will check and update.

Reply:
Yes, I overlooked during conflicts, I have added it now.
@goodov